azurerm/internal/services/videoanalyzer/resourceids.go

```diff
@@ -0,0 +1,3 @@
+package videoanalyzer
+
+//go:generate go run ../../tools/generator-resource-id/main.go -path=./ -name=VideoAnalyzer -id=/subscriptions/12345678-1234-9876-4563-123456789012/resourceGroups/resGroup1/providers/Microsoft.Media/videoanalyzers/analyzer1
```

Comment:
In ARM these are supposed to be camelCased for different words, since this is a new resource can we make this:

```suggestion
//go:generate go run ../../tools/generator-resource-id/main.go -path=./ -name=VideoAnalyzer -id=/subscriptions/12345678-1234-9876-4563-123456789012/resourceGroups/resGroup1/providers/Microsoft.Media/videoAnalyzers/analyzer1
```